api4/apitestlib.go

```diff
@@ -704,6 +704,22 @@ func (me *TestHelper) CreateGroup() *model.Group {
 	return group
 }
 
+// TestForClientAndLocal runs a test function for the normal client and as a local client.
+func (me *TestHelper) TestForClientAndLocal(t *testing.T, f func(*testing.T, *model.Client4), name ...string) {
```

Comment:
nonblocking: the local client actions are supposed to be closer to a sysadmin than to an unprivileged client, so it's difficult that this helper ends being used. I'm happy leaving it here tho 1/5

Reply:
Ah I see. That makes sense. Will remove.